README.md

```diff
@@ -68,11 +68,14 @@ Please read the [CONTRIBUTING.md](./CONTRIBUTING.md) for details on code of cond
 
 ## Communication
 
-Contact us through email: axon@axonweb3.io
+The following ways are a great spot to ask questions about Axon:
```

Comment:
If you have any questions about Axon, reach out to us via the following methods: